<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Bootstrap Support Values for Genome Distances</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Bootstrap Support Values for Genome Distances</h1>
      <h2 class="project-tagline"></h2>
      <!-- <a href="https://github.com/EvolBioInf/gcb2015" class="btn">View on GitHub</a>
      <a href="https://github.com/EvolBioInf/gcb2015/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/EvolBioInf/gcb2015/tarball/master" class="btn">Download .tar.gz</a> -->
    </section>

    <section class="main-content">

<h3><a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span class="octicon octicon-link"></span></a>Bootstrap Support Values for Genome Distances.</h3>

<p><b>Abstract:</b> We have recently developed and implemented a distance for efficiently estimating the number of
substitutions per site between unaligned genome sequences. These substitution rates are called
“anchor distances” and can be used for phylogeny reconstruction. Most phylogenies come with
bootstrap support values, which are computed by resampling with replacement columns of homolo-
gous residues from the original alignment. Unfortunately, this method cannot be applied to anchor
distances, as they are based on approximate pairwise local alignments rather than the full multiple
sequence alignment necessary for the classical bootstrap. As an alternative, we propose to boot-
strap pairs of genomes by resampling with replacement the local alignments on which the original
anchor distance is based. In this paper we present an implementation of this idea and demonstrate
its usefulness by comparing it to alignment-based bootstrap when applied to simulated and real
genome samples.</p>


<h3><a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span class="octicon octicon-link"></span></a>Data Sets.</h3>

<p>Here follow all the data sets used for evaluation in the paper.</p>

<ul>
  <li><a href="https://github.com/EvolBioInf/gcb2015/raw/gh-pages/eco29.fasta.gz">Eco29</a>: A set of 29 <i>E. coli / Shigella</i> genomes. <a href="https://github.com/EvolBioInf/gcb2015/raw/gh-pages/eco29mugsy.fasta.gz">The reference alignment.</a></li>
  <li><a href="https://github.com/EvolBioInf/gcb2015/raw/gh-pages/humanMito.tgz">Human mitochondrial genomes </a></li>
</ul>

<h3><a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span class="octicon octicon-link"></span></a>Programs and Tools.</h3>

<p>For the analysis of the support values we have written quite a number of programs. Also we used scripts to conveniently wrap other programs like <tt>phylip</tt>.</p>

<ul>
  <li><a href="https://github.com/EvolBioInf/gcb2015/raw/gh-pages/clustDist_0.7.tgz">clustDist</a>: Clusters sequences according to a distance matrix</li>
  <li><a href="https://github.com/EvolBioInf/gcb2015/raw/gh-pages/dnaDist_0.3.tgz">dnaDist</a>: Compute a distance matrix from an alignment</li>
  <li><a href="https://github.com/EvolBioInf/gcb2015/raw/gh-pages/senSpec_0.4.tgz">senSpec</a>: Given a true tree and an estimation, this program calculates the sensitivity and specificity.</li>
  <li><a href="https://github.com/EvolBioInf/andi/tree/bootstrap">andi (bootstrap branch)</a>: Efficiently estimates evolutionary distances</li>
  <li><a href="https://github.com/EvolBioInf/gcb2015/raw/gh-pages/joinBoot_0.1.tgz">joinBoot</a>: Compares the results from two bootstrap experiments.</li>
</ul>



      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/EvolBioInf/gcb2015">Gcb2015</a> is maintained by <a href="https://github.com/EvolBioInf">EvolBioInf</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>

